code/game/objects/structures/crates_lockers/closets/secure/cm_closets.dm

```diff
@@ -140,57 +148,60 @@
 	slotlocktype = "engi"
 	health = 100000 // just no
 
-	New()
-		..()
-		new /obj/item/storage/belt/utility/full(src)
-		new /obj/item/clothing/glasses/welding(src)
-		new /obj/item/clothing/tie/storage/webbing(src)
+/obj/structure/closet/secure_closet/marine/engi/New()
+	. = ..()
+	new /obj/item/storage/belt/utility/full(src)
+	new /obj/item/clothing/glasses/welding(src)
+	new /obj/item/clothing/tie/storage/webbing(src)
+
+/obj/structure/closet/secure_closet/marine/engi/select_gamemode_equipment()
```

Comment:
Same reason. Gamemode equipment should actually change based on gamemode, not be just a /New()